Virtual Librarian/VirtualLibrarian.API/Core/ILibrary.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VirtualLibrarian.Domain;
+
+namespace VirtualLibrarian.API.Core
+{
+    public interface ILibrary
+    {
+        void AddBook(Book book);
+        void AddBookCopy(BookCopy bookCopy, Book book);
+        void RemoveBook(Book book);
+        void RemoveBookCopy(BookCopy bookCopy, Book book);
+        List<Book> GetAvailableBooksSorted();
+        List<Book> GetAllBooks();
+        List<BookCopy> GetAvailableBookCopies();
+        List<User> GetAllUsers();
+        List<Admin> GetAllAdmins();
```

Comment:
Are you sure you want to expose this? :D

Reply:
Currently, it is unused functionality. If you are referring to GetAllAdmins, what would you do differently about it?